doc/lua_api.txt

```diff
@@ -2268,7 +2269,101 @@ Elements
 **Note**: do _not_ use a element name starting with `key_`; those names are
 reserved to pass key press events to formspec!
 
+### `style[<name>;<prop1>;<prop2>;...]`
 
+Set the style for the named element `name`.
+Note: this **must** be before the element is defined.
+
+See [Styling Formspecs].
+
+
+### `style_type[<type>;<prop1>;<prop2>;...]`
+
+Sets the style for all elements of type `type` which appear after this element.
+
+See [Styling Formspecs].
```

Comment:
- Asterisks are missing.
- The `**Note**: do _not_ use a element name starting with `key_`; those names are
reserved to pass key press events to formspec!` should probably be below all elements.